modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractGoCodegen.java

```diff
@@ -681,6 +681,11 @@ public ModelsMap postProcessModels(ModelsMap objs) {
                 imports.add(createMapping("import", "time"));
             }
 
+            // if allOf contains "time.Time" type
```

Comment:
This fixes the problem if allOf contain base field but there is one outstanding issue. If oneOf or allOf are object containing that field (inline) then import is missing. 

Somehow    `codegenProperties.addAll(inheritedProperties);` aren't including that properties so we do not detect it here but they are present in the template - thus compilation issue